### PR TITLE
remove unwanted data leak, relates to issue #168

### PR DESCRIPTION
**Pull request by kruisdraad**
see issue for details, this PR is a proposal for a fix. The DIV style could be better, however i am not that good with HTML/DIV codes ;)

#168 (ed: this is a link to the issue)

### CODE REVIEW

File: src/resources/views/inc/sidebar.blade.php [hunk 1]
```diff
@@ -6,11 +6,10 @@
         <!-- Sidebar user panel -->
         <div class="user-panel">
           <div class="pull-left image">
-            <img src="https://placehold.it/160x160/00a65a/ffffff/&text={{ mb_substr(Auth::user()->name, 0, 1) }}" class="img-circle" alt="User Image">
+            <br>
```
Comment (lloy0076) [inline src/resources/views/inc/sidebar.blade.php]: I agree, in principle, with removing the call to placehold.it but I'm not sure replacing it with a break is the best solution.

**Comment by tabacitu** [comment]
Hi @kruisdraad ,

Thanks a lot for the PR. As I mentioned in the issue, I don't think this is a security issue, or a leak. We only send one letter to Placehold.it - which I don't think is enough for _anything_ :-)

So in my opinion - not an issue. Better to have the placehold image. 
@lloy0076 , what do you think?

Cheers!

**Comment by lloy0076** [comment]
A config option with some transparent gif would address any leakage to "placehold.it".

On the other hand, a single letter isn't too grand a leak (although it is technically an information leak).

That said, I'm on a stupidly slow connection at the moment and it *is* just one more external call that needs to be made (where each call is hit by a minimum of 1000ms latency)...

@tabacitu - I'm happy either way!

**Comment by kruisdraad** [comment]
I also shows when people logged in and the refferer. It was very late when i saw the code, so i missed the 1 letter part, still for me i don't like it when every logging gets a call. Same reason i am localizing all the JS files on a static host ;)

**Comment by tabacitu** [comment]
Hmm... I think that's overkill, myself. I might be wrong - it's been known to happen :-) - but I think we're better off having the placehold image. Feel free to remove it from your own projects, though - totally understand.

@lloy0076 - magic happens when you move to HTTP/2 :-) You no longer have to worry about having too many JS/CSS files loading - it's actually better to have many files rather than one big file, because they load asynchronously. I always use the [free CloudFlare plan](https://www.cloudflare.com/) as my DNS - you get HTTP/2, excellent caching, better security and some other nice features too.

**Comment by tabacitu** [comment]
If anybody else stumbles upon this and feels I've been in the wrong, do tell :-)

**Pull request closed without merging**